packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -487,24 +512,30 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 		openChat();
 
 		streaming.value = true;
+		const payload: ChatRequest.RequestPayload['payload'] = {
+			role: 'user',
+			type: 'init-error-helper',
+			user: {
+				firstName: usersStore.currentUser?.firstName ?? '',
+			},
+			error: context.error,
+			node: assistantHelpers.processNodeForAssistant(context.node, [
+				'position',
+				'parameters.notice',
+			]),
+			nodeInputData,
+			executionSchema: schemas,
+			authType,
+		};
+		const payloadSize = getObjectSizeInKB(payload);
+		if (payloadSize > AI_ASSISTANT_MAX_CONTENT_LENGTH) {
+			// Remove parameters property from payload.node
+			payload.node.parameters = {};
+		}
 		chatWithAssistant(
```

Comment:
both functions updated here call `chatWithAssistant`? Can we do this check at a lower level to reduce duplication?

Reply:
I decided to leave it here since we have better type info at this level but might make sense once we introduce more checks as discussed above.